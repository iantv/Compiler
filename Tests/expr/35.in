a + b*c ? 123 - 123*0 : f()