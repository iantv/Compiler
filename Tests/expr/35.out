	()
		f
			0
		*
			123
	-
		123
?:
			c
		*
			b
	+
		a
